frontend/src/_helpers/appUtils.js

```diff
@@ -935,6 +935,12 @@ export function runQuery(_ref, queryId, queryName, confirmed = undefined, mode =
   const query = useDataQueriesStore.getState().dataQueries.find((query) => query.id === queryId);
   let dataQuery = {};
 
+  const { setPreviewLoading, setPreviewData } = useQueryPanelStore.getState().actions;
+  if (parameters?.shouldSetPreviewData) {
+    setPreviewLoading(true);
+    setPreviewData(null);
```

Comment:
set this to empty string (keep it according to the hotfix Nakul is pushing for showing falsy values)